core/src/main/scala/org/apache/spark/metrics/MetricsSystem.scala

```diff
@@ -155,6 +156,12 @@ private[spark] class MetricsSystem private (
     } else { defaultName }
   }
 
+  private[spark] def getRegisteredNames(source: Source): mutable.Set[String] = {
```

Comment:
Does not need to be mutable, or a Set. Just Seq?

Reply:
you are right. thanks for your advice.